packages/web-runtime/src/components/Topbar/ApplicationsMenu.vue

```diff
@@ -53,11 +53,12 @@ import { clientService } from 'web-pkg/src/services'
 import { configurationManager } from 'web-pkg/src/configuration'
 import { mapGetters } from 'vuex'
 import { urlJoin } from 'web-client/src/utils'
+import { PropType } from 'vue-demi'
```

Comment:
```suggestion
import { PropType } from 'vue'
```

 (probably merge with import above)